docs/README.md

```diff
@@ -25,6 +25,9 @@ user:
 news:
   title: Events & News
   content:
+  - header: Welcome Weiyi Liu as new committer
+    content: Based on his continuous contributions, Weiyi Liu (a.k.a [wayilau](https://github.com/wayilau)) has been voted as a new committer.
+    time: Dec. 10th 2019
```

Comment:
You should delete the oldest one :) We just keep 4 on the front page.

Reply:
ok.